vcluster/configure/vcluster-yaml/external/platform/api-key.mdx

```diff
@@ -2,28 +2,262 @@
+
+        <br />
+  <!-- vale off -->
+  <InterpolatedCodeBlock
+  code={
+  `kubectl create secret generic [[VAR:SECRET NAME:vcluster-platform-api-key]] \\
+    --from-literal=accessKey=[[VAR:ACCESS KEY:YOUR_ACCESS_KEY]] \\
+    --from-literal=host=[[VAR:PLATFORM HOST:YOUR_PLATFORM_HOST]] \\
+    -n [[VAR:SECRET NAMESPACE:SECRET_NAMESPACE]]`}
+          language="bash"
+          title="Create Secret"
+  />
+  <!-- vale on -->
+
+      </Step>
+      
+      <Step title="Reference the Secret in vcluster.yaml">
+        
+        In your `vcluster.yaml` file, define the following parameters to reference the Secret and configure access control:
+
+        - `secretName`: Specifies the name of the Secret containing the API key. Defaults to `vcluster-platform-api-key` if undefined.
+        - `namespace`: Specifies the namespace where the Secret is located. If different from the virtual cluster's namespace, ensure that the virtual cluster has access to this namespace.
+        - `createRBAC`: When set to `true`, vCluster automatically creates the necessary `Role` and `RoleBinding` resources to allow access to the Secret. Defaults to true.
+
+        <br />
+        
+  <!-- vale off -->
+  <InterpolatedCodeBlock
+    code={
+  `external:
+    platform:
+      apiKey:
+        secretName: [[VAR:SECRET NAME:vcluster-platform-api-key]]
+        namespace: [[VAR:NAMESPACE:VCLUSTER_NAMESPACE]]
+        createRBAC: [[VAR:CREATE RBAC:true]]
+  `}
+      language="yaml"
+        />       
+      </Step>
+    </Flow>
+  </TabItem>
+  
+  <TabItem value="individual" label="Individual user access">
+
+  ### Connect a virtual cluster to the platform with an individual user access key
+
+  Authenticate a virtual cluster with the platform using a personal access key.
+
+  ### Prerequisites
+
+  Before you begin, ensure you have: 
+
+  - Access to the vCluster Platform.
+  - The [vCluster CLI installed](/platform/install/quick-start-guide) installed on your local machine.
+  - The ability for each user to generate their own access key from the platform console.
+    
+  ### Connect a virtual cluster to the platform using individual user access
+
+  <Flow>
+    <Step title="Locate or create the vcluster.yaml file">
+
+    You must configure `vcluster.yaml` to define how your virtual cluster connects to the platform.
+
+    - If you are creating a new virtual cluster, create a new `vcluster.yaml` file in your working directory.
+    - If you are modifying an existing virtual cluster, locate the `vcluster.yaml` file used during its deployment.
+
+    :::note
+    The `vcluster.yaml` file is typically stored locally in your project folder or provided as part of the Helm configuration used to deploy the virtual cluster.
+    :::
+
+    </Step>
+
+    <Step title="Configure and deploy vcluster.yaml for individual user access">
+
+    Update your `vcluster.yaml` file to enable individual user authentication:
```

Comment:
And here we are also saying individual **user**, but IMO it should be individual virtual cluster